### ISSUE TITLE
BUG: All tests failing - HTTPoison.Error{...}

### ISSUE THREAD

**Issue by scottswezey** [scottswezey/twilio_magic_values#13]
**Describe the bug**
All tests fail with the same `HTTPoison.Error{id: nil, reason: {:option, :server_only, :honor_cipher_order}`

**To Reproduce**

1. Run `mix test`

**Versions**

- Elixir: 1.9.1
- Erlang: 22
- Twilio_Magic_Values: 1.0.3
- Twilio API: ???

**Additional context**

```
 43) test cant_receive_sms (TwilioMagicValues.MessagingTest)
     test/twilio_magic_values/messaging_test.exs:105
     ** (MatchError) no match of right hand side value: {:error, %HTTPoison.Error{id: nil, reason: {:option, :server_only, :honor_cipher_order}}}
     code: send_message(
     stacktrace:
       test/twilio_magic_values/messaging_test.exs:137: TwilioMagicValues.MessagingTest.get_http_response/1
       test/twilio_magic_values/messaging_test.exs:125: TwilioMagicValues.MessagingTest.send_message/1
       test/twilio_magic_values/messaging_test.exs:107: (test)
```

**Comment by scottswezey** [comment]
See related issue https://github.com/benoitc/hackney/issues/591

**Comment by scottswezey** [comment]
I believe this should be resolved when updating to HTTPoison 1.6+ or hackney 1.15.2.

`mix deps.update hackney` or `mix deps.update httpoison`
